### ISSUE TITLE
Combination of Cross-View communication and AngularTouch/AngularGestures fails

### ISSUE THREAD

**Issue by Rajan** [AppGyver/supersonic-kitchensink#9]
The variable appears as it is when we try to include the ngTouch in the module.

<h3>write here text</h3>

    <div class="button button-block button-positive" ng-swipe-right="swiped();">
        Swipe
    </div>

    <input ng-model="msg1"/>

    <h3>you wrote</h3>

    <h3>
        {{msg1}}
    </h3>

Here's the module code.

angular
  .module('example', ['angular-gestures'])
    .controller('LearnMoreController', function($scope, supersonic) {

    $scope.navbarTitle = "Learn More";
    $scope.swiped = function (){
        supersonic.ui.dialog.alert('swiped');
    };
    $scope.msg1 = "";

    // bind it to superscope
    supersonic.bind($scope, "msg1");

  });

**Comment by harsaharsa** [comment]
Have you included ng-touch as a dependency via Bower? It's not included by default.

**Comment by Rajan** [comment]
Yes. I did try that.. but the ng-click stopped working after that.
We probably should include ng-touch and swipe in the KitchenSink app.
